src/app/profiles/store/sagas/locationChange.saga.ts

```diff
@@ -17,10 +17,28 @@ function* contributorLocationSaga(match: Match<{ id: string }>) {
 }
 
 export default function* locationChangeSaga() {
-  yield takeLatestLocationChange(CONTRIBUTORS_PATH, contributorsLocationSaga);
-  yield takeLatestLocationChange('/mes-abonnements', contributorsLocationSaga);
+  yield takeLatestLocationChange(
```

Comment:
Others suggestions might help to simplify this saga.
If they don't try to  implement a version of `takeLatestLocationChange` which could take an array as first parameter.